homeassistant/components/logbook/__init__.py

```diff
@@ -357,23 +336,7 @@ def _get_related_entity_ids(session, entity_filter):
 
 
 def _generate_filter_from_config(config):
-    excluded_entities = []
-    excluded_domains = []
-    included_entities = []
-    included_domains = []
-
-    exclude = config.get(CONF_EXCLUDE)
-    if exclude:
-        excluded_entities = exclude.get(CONF_ENTITIES, [])
-        excluded_domains = exclude.get(CONF_DOMAINS, [])
-    include = config.get(CONF_INCLUDE)
-    if include:
-        included_entities = include.get(CONF_ENTITIES, [])
-        included_domains = include.get(CONF_DOMAINS, [])
-
-    return generate_filter(
-        included_domains, included_entities, excluded_domains, excluded_entities
-    )
+    return convert_filter_alt(config)
```

Comment:
We should call the function (when we settle on a name) directly here and in the tests instead of increasing the stack.

Reply:
This one is good now I believe, the `_generate_filter_from_config` method is removed from everywhere.